### PR TITLE
Forms - Cards button is always blue

### PR DESCRIPTION
**Pull request by jsondoo**
After today's user testing with @xtinah-w, the Edit Resource button being black was confusing and unfriendly so we change it to be always blue so that it looks active.

**How to test**
confirm that the button is always blue

alwaysBlue

### CODE REVIEW

**Comment by SahilTara** [comment]
Ah yeah guessing people thought it was disabled or something?

**Review by SahilTara — DISMISSED** [review]

**Comment by jsondoo** [comment]
@SahilTara yea, did you guys have a reason for changing the color for edit resource to black?

**Comment by SahilTara** [comment]
@jsondoo think it was just a secondary colour not sure, notice how our back button is also similar?

File: src/client/src/translations/whitelist_en.json [hunk 1]
```diff
@@ -16,7 +16,6 @@
   "summaryTile.changeItemName",
   "topNavBar.ariaNavLabel",
   "topNavBar.frameworks",
-  "topNavBar.pages",
```
Comment (jcoc611-microsoft) [inline src/client/src/translations/whitelist_en.json]: This diff comes from changes done in a different PR right? Maybe I'm misunderstanding something, but if so then let's try to keep the translations changes on the PR that actually changes them.

**Review by jcoc611-microsoft — DISMISSED** [review]
nice! UX getting better every day

File: src/client/src/containers/AzureSubscriptions/index.tsx [hunk 2]
```diff
@@ -129,7 +129,7 @@ class AzureSubscriptions extends React.Component<Props, IState> {
               )}
               disabled={!isLoggedIn}
               handleDetailsClick={setDetailPage}
-              useNormalButtons={this.isSelectionCreated(option.internalName)}
+              useNormalButtons={false}
```
Comment (jcoc611-microsoft) [inline src/client/src/containers/AzureSubscriptions/index.tsx]: lol what are *normal* buttons?

Comment (SahilTara) [reply src/client/src/containers/AzureSubscriptions/index.tsx]: Unhighlighted buttons, the dark buttons are "normal buttons"

**Review by jasmingoh — DISMISSED** [review]

**Comment by ngkelly3** [comment]
> @SahilTara yea, did you guys have a reason for changing the color for edit resource to black?

Blue was to signify a call to action from what I remember, so to help guide the developer through the wizard. If a selection was made then the button turned black signifying that they already completed an action there.

File: src/client/src/containers/AzureSubscriptions/index.tsx (hunk 2 above)
Comment (ngkelly3) [reply src/client/src/containers/AzureSubscriptions/index.tsx]: If they're always going to be blue from now on, does this prop need to be here?

**Review by streetcoder123 — APPROVED** [review]

**Review by jasmingoh — APPROVED** [review]

**Review by SahilTara — APPROVED** [review]
